parlai/crowdsourcing/tasks/pairwise_per_turn_eval/README.md

```diff
@@ -0,0 +1,29 @@
+# Per-turn Evaluation Crowdsourcing Task
+In this task, Turkers compare two models on every turn of a conversation by choosing the response based on a certain criteria. Each time the bot responds, it will give two possible responses instead of just one. The Turker will choose which of the two responses is better according to some metric (see here for a list), provide a justification of why the Turker chose that response, and then that choice will be accepted as the bot’s response and the conversation will continue. 
```

Comment:
Maybe "Each time the Turker's partner responds, the Turker will see two possible responses, one from each of two models." for clarity?

Reply:
Also, "see here for a list" doesn't link to anything